backend/corpora/common/entities/collection.py

```diff
@@ -254,19 +326,19 @@ def tombstone_collection(self):
             ds.tombstone_dataset_and_delete_child_objects()
         self.session.commit()
 
-    def update(self, links: list = None, **kwargs) -> None:
+    def update(self, links: list = None, keep_links=None, **kwargs) -> None:
```

Comment:
How about this:

```
def update(self, links: list = None, **kwargs) -> None:
        if isinstance(links, list):
            for link in self.links:
                self.session.delete(link)
       new_objs = [DbCollectionLink(collection_id=self.id, **link) for link in links]

Reply:
I think this would require more work to make sure that in cases where we want to keep the links as they are (without modifying timestamps), then we'd have to make sure to remove `links` from the arguments that are passed in.